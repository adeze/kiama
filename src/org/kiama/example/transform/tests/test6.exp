op + 2
op * 2
var x
var y
var x
y * x * 3
