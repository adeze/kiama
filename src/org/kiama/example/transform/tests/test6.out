Program(Map(+ -> 2, * -> 2),List(VarDecl(x), VarDecl(y), VarDecl(x)),BinExpR(Var(y),*,BinExpR(Var(x),*,Factor(Num(3)))))
BinExp(BinExp(Var(y),*,Var(x)),*,Num(3))
